components/frontend/package-lock.json

```diff
@@ -8133,7 +8133,9 @@
             "license": "MIT"
         },
         "node_modules/cookie": {
-            "version": "0.6.0",
+            "version": "0.7.1",
```

Comment:
Are these additional changes, or were they needed for uv?

Reply:
These were skipped before because I forgot to add `--upgrade` to `uv pip compile`.